benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/models/BenchFlowTestNumber.java

```diff
@@ -14,7 +14,7 @@
 
   @Id private String testIdentifier;
 
-  private Long counter = 0L;
+  private Long counter = 1L;
```

Comment:
In the constructor you can see that it takes a a `userName` and `testName` as input meaning that there will be one `BenchFlowTestNumber` object stored in the DB per combination. In the [TestModelDAO](https://github.com/benchflow/benchflow/blob/408297ca24ea6f12d257f9c4ca33b55ea3e236ef/benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/services/internal/dao/BenchFlowTestModelDAO.java#L43) class you can see how the test number is generated.